ql/src/test/results/clientpositive/llap/lateral_view_onview2.q.out

```diff
@@ -52,6 +52,8 @@ STAGE PLANS:
       Processor Tree:
         TableScan
           alias: lv_table_n1
+          properties:
+            insideView TRUE
```

Comment:
Why is this new property appeared here?

Reply:
I think this is a result of using CalcitePlanner.  The query here contains a view, so this seems correct to me.